pyEpiabm/pyEpiabm/tests/test_toy_population_config.py

```diff
@@ -0,0 +1,77 @@
+import unittest
+import pyEpiabm as pe
+from pyEpiabm.toy_population_config import ToyPopulation
+
+
+class TestTopPopConfig(unittest.TestCase):
+    """Test the Toy Population class.
+    """
+    @classmethod
+    def setUpClass(cls) -> None:
```

Comment:
How sensitive is this test to the initial parameter values? Could we even set it up to run over a random collection of parameters? (examples of this in `seirmo/test_gillespie`)

Reply:
Not sure in what context you're using sensitive. This just takes n people and distributes them among some given amount of cells/microcells/households. Its a multinomial with equal probs so can do some stats analysis on the distribution of people across microcells if you want (as pop gets large it gets more uniform - CLT so might need to introduce artificial variation). But if you mean how it affects disease progression, probably only that with very few people you amplify individualistic effects. If you mean we should put random number sin where we've got a lot of 5s and 1000, maybe, but there's some physical common sense here, like you don't want more microcells households than people so I'm not sure it would be useful.

Reply:
Just really in reference to the previous comment, that currently the tests could fail (or at least their performance changes) when the population size is varied. Similarly, there could be odd behaviour when `cell_number != microcell_per_cell` but we don't test for this. I think it's always good practice to test for any case we might want to use, so we could require that the number of microcells < population, but test random combinations that satisfy this (or 10 * number of microcells < population, or whatever we might want to require). As much as anything, it would give a good idea of when our code breaks down (if ever), for example when applying to different population data (such as very dilute populations)